tests/flow/test_persistency.py

```diff
@@ -128,7 +128,7 @@ def test01_save_load(self):
                 indices = graph.query("""CALL db.indexes()""").result_set
                 expected_indices = {
                         'country': [['name', 'population'], 'english', [], 'NODE'],
-                        'person': [['name', 'height', 'text', 'embedding'], OrderedDict({'name': ['RANGE'], 'height': ['RANGE'], 'text': ['FULLTEXT'], 'embedding': ['VECTOR']}), OrderedDict({'name': OrderedDict({}), 'height': OrderedDict({}), 'text': OrderedDict({}), 'embedding': OrderedDict({'dimension': 128, 'M': 64, 'efConstruction': 10, 'efRuntime': 10})}), 'english', ['a', 'b'], 'NODE'],
+                        'person': [['name', 'height', 'text', 'embedding'], OrderedDict({'name': ['RANGE'], 'height': ['RANGE'], 'text': ['FULLTEXT'], 'embedding': ['VECTOR']}), OrderedDict({'name': OrderedDict({}), 'height': OrderedDict({}), 'text': OrderedDict({}), 'embedding': OrderedDict({'dimension': 128, 'similarityFunction': 'euclidean', 'M': 64, 'efConstruction': 10, 'efRuntime': 10})}), 'english', ['a', 'b'], 'NODE'],
```

Comment:
Please add a test, not necessarily in this file, which checks a vector index with the new metric function.